sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -1740,6 +1741,57 @@ def PatchItem(self, document_link, operations, options=None, **kwargs):
         self._UpdateSessionIfRequired(headers, result, self.last_response_headers)
         return result
 
+    def Batch(self, collection_link, batch_operations, options=None, **kwargs):
+        """Executes the given operations in transactional batch.
+
+        :param str collection_link: The link to the collection
+        :param list batch_operations: The batch of operations for the batch request.
+        :param dict options: The request options for the request.
+
+        :return:
+            The result of the batch operation.
+        :rtype:
+            dict
+
+        """
+        if options is None:
+            options = {}
+
+        path = base.GetPathFromLink(collection_link, "docs")
+        collection_id = base.GetResourceIdOrFullNameFromLink(collection_link)
+
+        batch_operations = base._format_batch_operations(batch_operations)
+
+        result, self.last_response_headers = self._Batch(
+            batch_operations,
+            path,
+            collection_id,
+            options,
+            **kwargs)
+
+        final_responses = []
+        is_error = False
```

Comment:
if there is no results being returned (not sure whether this will ever gonna happen), then we will return a response with is_error: false, empty batch operations

I see in Java SDK, there is one more check to compare the result size with the input operation size, if mismatch, then throw internalServerError exceptions.
https://github.com/Azure/azure-sdk-for-java/blob/07f29bfd599460dfa7071f4f1f3de41257db28df/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BatchResponseParser.java#L75